src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -104,6 +104,19 @@
             <property name="table_name_prefix" value="sample"/>
         </include>
     </select>
+
+
+    <!-- for /molecular-profile-sample-counts/fetch (returns GenomicDataCount) -->
+    <select id="getGenomicDataCounts" resultType="org.cbioportal.model.GenomicDataCount">
+        SELECT genetic_profile.stable_id AS value, genetic_profile.stable_id AS value2, genetic_profile.name AS label, count(sample_profile.genetic_profile_id) AS count FROM sample_profile
```

Comment:
Can we have each column on a new line? (just easier to read)
Ex. 
Select 
  stable_id,
  name